### ISSUE TITLE
check on which fields in the development projects csv urbansim really needs 

### ISSUE THREAD

**Issue by tbuckl** [bobbylu/building_projects#5]
the development_projects.csv is read in here:

https://github.com/MetropolitanTransportationCommission/bayarea_urbansim/blob/master/baus/datasources.py#L191-L197

apparently the parcels are just marked by the geom_id and then developed like that. 

so perhaps a minimal version of this csv would be just the `geom_id` column? 

i can ask @fscottfoti when i get the chance. if so, this narrows the problem down a bit.

**Comment by tbuckl** [comment]
nevermind, i spoke too soon. the csv is read in at 2 other places as well:

https://github.com/MetropolitanTransportationCommission/bayarea_urbansim/blob/master/baus/datasources.py#L252-L323

**Comment by fscottfoti** [comment]
Yeah sorry about that.  It is a read a few times cause it only takes a fraction of a second because the file is small.  It could be refactored to only be read in once if you wanted.

A minimal version of what csv - the development projects list?  It needs other columns besides geom_id - mainly those that are in the buildings table - the ones used on this line and the next few:

https://github.com/MetropolitanTransportationCommission/bayarea_urbansim/blob/master/baus/datasources.py#L293

building_type
stories
building_sqft
non_residential_sqft
residential_units
year_built

maybe that's it?

**Comment by tombuckley** [comment]
thanks @fscottfoti this is helpful

**Comment by tbuckl** [comment]
@bobbylu if you're looking for a next step here, i'd take a look at @fscottfoti's comment

**Comment by tbuckl** [comment]
@bobbylu i think the first finished task here is for you to submit a pull request to https://github.com/MetropolitanTransportationCommission/bayarea_urbansim/ with 1 new project/building in the development_projects.csv. feel free to ask questions!

**Comment by bobbylu** [comment]
@tombuckley gotcha! /giphy thumbup

**Comment by tbuckl** [comment]
@bobbylu you could even try making up fake values for things you don't know.

**Comment by tbuckl** [comment]
@bobbylu i think you should also be able to fork the repository

https://github.com/MetropolitanTransportationCommission/bayarea_urbansim/

https://help.github.com/articles/fork-a-repo/ 

so you could try forking it to your account, doing the work on the buildings project on a branch, as we did yesterday, and then issuing the pull request to the repository on your account first, if you want to test it.

**Comment by bobbylu** [comment]
@tombuckley yes, sir!

**Comment by bobbylu** [comment]
@tombuckley so I forked it (cloned it) to my github respo, I made commits and created a pull request, but then the request was pushed to MTC's respo, instead of mine....should this be happening?

I don't know how to make csv changes without the desktop thing, because I am editing the file in the respo, I had to download it. But then if I don't use the desktop Github thing, I don't know how to actually made a commit. Should I just upload the new file and delete the old one? I don't think so.

We should probably discuss this tomorrow, maybe you can teach me how to use the command line functions.

**Comment by tbuckl** [comment]
@bobbylu ill try to run a simulation with your changes and we can chat more tomorrow in person about all the above

**Comment by bobbylu** [comment]
🏄
